test/helpers.bash

```diff
@@ -36,6 +36,7 @@ RUNTIME_NAME=${RUNTIME_NAME:-runc}
 CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-runc}
 CONTAINER_UID_MAPPINGS=${CONTAINER_UID_MAPPINGS:-}
 CONTAINER_GID_MAPPINGS=${CONTAINER_GID_MAPPINGS:-}
+CONTAINER_MANAGE_NS_LIFECYCLE=${CONTAINER_MANAGE_NS_LIFECYCLE:-true}
```

Comment:
So I assume that the new code is now integration tested as well and we dont have to add more, right?

Reply:
yeah it's being tested both in integration and e2e. I intend on adding more tests in the future, but figure we can get this in for now